### ISSUE TITLE
WYSIWYG Editor missing

### ISSUE THREAD

**Issue by wtl420** [Jaxboards/Jaxboards#22]
I removed [this file](https://github.com/seanjohnson08/JXBD_Git/blob/master/wysiwyg.php) early on, my guess is that it's needed for it. I tried restoring the file but it still didn't show up. Wasn't able to find a reference to the file in the code either, not sure what the secret sauce is here (though it's entirely possible I occidentally removed something necessary for it).

**Comment by seanjohnson08** [comment]
That file was just a way for me to demo/work on it outside of jaxboards, and isn't needed (as you surmised). Given how complex the wysiwyg editor is/was, my guess is I probably broke it in the JS refactor.

**Comment by wtl420** [comment]
Nah, I'm pretty sure I broke it somewhere along the line, not sure where though. I'm sure we can fix it back up though.
